src/modules.mjs

```diff
@@ -4,14 +4,29 @@
  * SPDX-License-Identifier: MIT
  */
 
-// "use strict";
+import * as path from 'path';
+
+import {
+  isDecorator,
+  assert,
+  isJsOnlySymbol,
+  error,
+  read,
+  warn,
+  setCurrentFile,
+  printErr,
+  addMacros,
+  runInMacroContext,
+  mergeInto,
+} from './utility.mjs';
```

Comment:
Could these long lists be `import *`?

Reply:
The problem is that if you switch to `*` then you also need to qualify all of the call sites.

e.g. `mergeInto` becomes `utils.mergeInto`.  

I think this would make the diff a lot larger, so maybe will do that as a followup?

Reply:
I see. Well, maybe `import *` is not worth it then, if it causes all those prefixes.

Reply:
I may make a followup with that change.